### PR TITLE
✨ Add Kubernetes Inventory pack

### PR DESCRIPTION
**Pull request by czunker**
Signed-off-by: Christian Zunker <christian@mondoo.com>

### CODE REVIEW

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          k8s.statefulset { * }
+  - uid: mondoo-kubernetes-replicasets-inventory
+    name: Mondoo Kubernetes ReplicaSets Inventory
+    filters:
+      - asset.platform == "k8s-replicasetset"
+    queries:
+      - uid: k8s-replicasetset
+        title: Gather ReplicaSet Information
+        query: |
+          k8s.replicasetset { * }
```
Comment (chris-rock) [inline core/mondoo-kubernetes-inventory.mql.yaml:95]: this needs to be `k8s.replicaset { * }`

**Review by chris-rock — COMMENTED** [review]
Great addition @czunker Once the pack is updated, LGTM

File: core/mondoo-kubernetes-inventory.mql.yaml (hunk 1 above)
Comment (czunker) [reply core/mondoo-kubernetes-inventory.mql.yaml]: Oh. Good catch. Thanks. Is fixed now.

**Review by chris-rock — APPROVED** [review]
Thank you @czunker Great addition

**Pull request merged**